src/core/a-scene.js

```diff
@@ -265,20 +260,30 @@ var AScene = module.exports = registerElement('a-scene', {
     /**
      * Notify scene that camera has been added and to remove the default.
      *
-e    * @param {object} el - element holding the camera component.
+     * @param {object} el - element holding the camera component.
      */
-    registerCamera: {
+    setActiveCamera: {
```

Comment:
would be nice later to move this into a separate component. change the active camera by changing the component

```
<a-scene camera="active: #orthographic"></a-scene>
```